app/common/templates/common/base.html

```diff
@@ -19,7 +19,7 @@
     "productName": "MHCLG Funding Service",
     "homepageUrl": "#",
     "classes": "govuk-header--full-width-border app-!-no-wrap",
-    "navigation": [{ "text": ("Sign in"), "href": url_for('auth.request_a_link_to_sign_in') }],
+    "navigation": [{ "text": ("Sign out"), "href": url_for('auth.sign_out') }] if current_user.is_authenticated else [],
```

Comment:
I think the proto/ design I last saw had a border but this could all be cleaned up/ aligned at the end?

Is current_user always set because flask login puts something on the global context or should this be checking it exists before trying to access `is_authenticated`? (Jinja might do that safely anyway, not sure!)

Reply:
yep current_user is always available, even before logging in